create_network_data.py

```diff
@@ -107,15 +111,16 @@ def main():
     #arcpy.FeatureClassToFeatureClass_conversion(bike_ped_auto, fgdb_dataset_name, 'AutoNetwork', "AutoNetwork = 'Y'")
 
     #: Export the BikePedAuto to shapefile
-    arcpy.FeatureClassToFeatureClass_conversion(bike_ped_auto, 'C:\\Users\\gbunce\\Documents\\projects\\MultimodalNetwork', 'BikePedAuto_' + strDate + '.shp')
+    arcpy.FeatureClassToFeatureClass_conversion(bike_ped_auto, r'C:\Multimodal Network Data', 'BikePedAuto_' + strDate + '.shp')
 
     print("create_network_data.py script is done!")
 
 
 # this function imports the user-defined utrans roads into the the netork dataset feature class 
 def import_RoadsIntoNetworkDataset(utrans_roads_to_import):
     # get # Get the dictionary of codes and descriptions for cartocode field, for transfering descriptions to RoadClass network field
-    gdb = 'C:\\Users\\gbunce\AppData\\Roaming\\ESRI\ArcGISPro\\Favorites\\internal@SGID@internal.agrc.utah.gov.sde'
+    # gdb = 'C:\\Users\\gbunce\AppData\\Roaming\\ESRI\ArcGISPro\\Favorites\\internal@SGID@internal.agrc.utah.gov.sde'
+    gdb = r'C:\Users\emneemann\AppData\Roaming\Esri\ArcGISPro\Favorites\internal.agrc.utah.gov (2).sde'
```

Comment:
you don't want to put this in a common location that would be accessible to everyone?

Reply:
I was wondering about that too, but unless we use the L:Drive SDE connections, we just repoint to our local favorites.